packages/kit/types/ambient.d.ts

```diff
@@ -113,10 +113,22 @@ declare module '$app/navigation' {
 		opts?: { replaceState?: boolean; noscroll?: boolean; keepfocus?: boolean; state?: any }
 	): Promise<void>;
 	/**
-	 * Causes any `load` functions belonging to the currently active page to re-run if they `fetch` the resource in question. If no argument is given, all resources will be invalidated. Returns a `Promise` that resolves when the page is subsequently updated.
+	 * Causes any `load` functions belonging to the currently active page to re-run if they registered it with `fetch` or `depends`. Returns a `Promise` that resolves when the page is subsequently updated.
```

Comment:
I think some words got lost in this sentence during editing 😛 

```suggestion
	 * Causes any `load` functions belonging to the currently active page to re-run if they depend on the resource in question, via `fetch` or `depends`. Returns a `Promise` that resolves when the page is subsequently updated.
```